x/emissions/module/abci.go

```diff
@@ -43,13 +43,6 @@ func EndBlocker(ctx context.Context, am AppModule) error {
 	}
 	sdkCtx.Logger().Debug(fmt.Sprintf("ABCI EndBlocker %d: Total Revenue: %v, Sum Weight: %v", blockHeight, totalRevenue, sumWeight))
 
-	// REWARDS (will internally filter any non-RewardReady topics)
-	err = rewards.EmitRewards(sdkCtx, am.keeper, blockHeight, weights, sumWeight, totalRevenue)
-	if err != nil {
-		sdkCtx.Logger().Error("Error calculating global emission per topic: ", err)
-		return errors.Wrapf(err, "Rewards error")
-	}
-
 	err = rewards.PickChurnableActiveTopics(
```

Comment:
Should rename this to be `UpdateNoncesOfActiveTopics` and update the comments atop its declaration.